cider/exec/processor/StatelessProcessor.cpp

```diff
@@ -19,24 +19,24 @@
  * under the License.
  */
 
-#include "StatelessProcessor.h"
+#include "exec/processor/StatelessProcessor.h"
 
-namespace cider::processor {
+namespace cider::exec::processor {
 
-StatelessProcessor::StatelessProcessor(const plan::SubstraitPlanPtr& plan,
-                                       const BatchProcessorContextPtr& context)
-    : DefaultBatchProcessor(plan, context) {}
-
-std::shared_ptr<CiderBatch> StatelessProcessor::getResult() {
-  if (!inputBatch_) {
+std::pair<struct ArrowArray*, struct ArrowSchema*> StatelessProcessor::getResult() {
```

Comment:
As mentioned above, more prefer to change this API as ```void getResult(ArrowArray&, ArrowSchema&)```, so we could directly use ```move``` method of output batch.